pages/glossary/modal-dialog.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Modal dialog
+key: modal-dialog
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+A modal dialog is a type of user interface element that appears on top of the main content of a [html web page][] or application and requires the user to take some action or make a decision before they can continue interacting with the program.
```

Comment:
I don't think "type of user interface" is objective.
Same for "main content". 
What is the "program" you mention?

I think this needs to focus on the element preventing interaction with all other elements that are not their children. 

This definition really needs to be objective, because it is used in the applicability of the rule.